jupyter_console/ptshell.py

```diff
@@ -355,8 +374,11 @@ def init_prompt_toolkit_cli(self):
         if self.simple_prompt or ('JUPYTER_CONSOLE_TEST' in os.environ):
             # Simple restricted interface for tests so we can find prompts with
             # pexpect. Multi-line input not supported.
+            @asyncio.coroutine
             def prompt():
-                return cast_unicode_py2(input('In [%d]: ' % self.execution_count))
+                prompt = 'In [%d]: ' % self.execution_count
+                raw = yield from async_input(prompt)
+                return cast_unicode_py2(raw)
```

Comment:
`cast_unicode_py2` is no longer required since #201 was merged

Reply:
Good point, I had to remove it when rebasing as the other ones

Reply:
61e479da89

Reply:
I ping you during the week for the tests.